test/com.cooltt

```diff
@@ -1,20 +1,20 @@
 def mycoe/fun : {
-  (A : (i : dim) -> univ) (B : (i: dim) -> univ)
-  (coe/A : (r : dim) (x : A r) (i : dim) -> sub {A i} {i=r} x)
-  (coe/B : (r : dim) (x : B r) (i : dim) -> sub {B i} {i=r} x)
-  (r : dim) (f : (_ : A r) -> B r) (i : dim) ->
+  (A : (i : 𝕀) -> univ) (B : (i: 𝕀) -> univ)
+  (coe/A : (r : 𝕀) (x : A r) (i : 𝕀) -> sub {A i} {i=r} x)
+  (coe/B : (r : 𝕀) (x : B r) (i : 𝕀) -> sub {B i} {i=r} x)
+  (r : 𝕀) (f : (_ : A r) -> B r) (i : 𝕀) ->
   sub {(_ : A i) -> B i} {i=r} f
 } = {
  \A B coe/A coe/B r f i x =>
```

Comment:
btw how do people feel about ⇒...

Reply:
It's fine with me. Btw the only thing I really kind of hate is the unicode lambda, but I am not sure what alternative there is.

This is honestly the main reason I didn't start 100% unicode, because I didn't want to answer the question.

Reply:
Yeah, I'm certainly for  ⇒ as a synonym of =>, but I don't really like => itself.

Reply:
How about infix lambda `{x ↦ x}`.

Reply:
I think it leads to grammatical issues, and I also find it kind of ugly.